packages/features/landing_page_screen/lib/src/landing_page.dart

```diff
@@ -22,8 +22,10 @@ class _LandingPageState extends ConsumerState<LandingPage> {
   late IFileService fileService;
   late IImageService imageService;
 
-  Future<List<Project>> _getProjects() async =>
-      database.projectDAO.getProjects();
+  Future<List<Project>> _getProjects() async {
+    await Future.delayed(const Duration(milliseconds: 500));
```

Comment:
This looks like a workaround solution. Are there other possible approaches?

Reply:
Yeah I worked on different approaches too
1) Use of AnimatedSwitcher
This is too uses time duration delay so would have been the same thing 
<img width="681" alt="image" src="https://github.com/Catrobat/Paintroid-Flutter/assets/78733360/9a1b6879-1e44-477b-81e2-14024825e648">

2) Checking of database connection outside of Futurebuilder 
This would increase the loading time when there are a good number of substantial projects, not worth it acc to me when the reward is just the same animation.